src/interfaces/UIIcon.ts

```diff
@@ -0,0 +1,3 @@
+import { Scale as AllScales} from "./common";
+
+export type Scale = Extract<AllScales, "s" | "m" | "l">
```

Comment:
For us TS noobs what is the difference between `Extract` and `Pick`. Is there an advantage to using extract here?